ghcup

```diff
@@ -898,15 +910,20 @@ compile_ghc() {
         edo tar -xf ghc-*-src.tar.xz
         edo cd "ghc-${myghcver}"
 
-        cat <<-EOF > mk/build.mk || die
-			BuildFlavour = quick
-			V=0
-			BUILD_MAN = NO
-			BUILD_SPHINX_HTML = NO
-			BUILD_SPHINX_PDF = NO
-			HADDOCK_DOCS = YES
-			GhcWithLlvmCodeGen = YES
-			EOF
+        if [ -n "${build_config}" ] ; then
+            edo cat "${build_config}" > mk/build.mk
+        else
+			cat <<-EOF > mk/build.mk || die
+				BuildFlavour = quick
```

Comment:
btw, `BuildFlavour = quick`  won't have any effect, that's not how that var works... (nor would it be a good idea to use that build-profile even if it worked)